src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/conversion.cpp

```diff
@@ -63,7 +63,17 @@ void ConvertCPULayerTest::SetUp() {
     auto primitive = selectedType;
     if (primitive.empty())
         primitive = getPrimitiveType();
-    if (!isInOutPrecisionSupported(inPrc, outPrc))
+#if defined(OPENVINO_ARCH_ARM64)
+    if (inPrc == ov::element::u4 || inPrc == ov::element::i4)
+        primitive = "ref";
+    else if (shapes.first.is_static() && shapes.first.rank().get_length() <= 6 &&
```

Comment:
Why do you have to add rank limitation for snippets? We support any rank 🤔

Reply:
Thanks Alexandra for the comment! As we have constrants on rank for tokenization https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp#L1055, so I followed this condition in test cases. Please feel free to have further discussions.

Reply:
Applied. Thanks Alexandra!